js/pages/iptt_report/components/report/tableBody.js

```diff
@@ -6,7 +6,7 @@ import { LevelGroup, IndicatorRow } from './tableRows';
 const ReportTableBody = inject('rootStore', 'filterStore')(
     observer(({ rootStore, filterStore }) => {
         return (
-        <tbody>
+        <tbody className={filterStore._shade ? filterStore._shade : null}>
```

Comment:
you don't have to change it, but I just wanna make sure you've seen the new optional chain operator in ES6 - you can write this whole line as `filterStore?._shade` and that does the same thing as the ternary you wrote out (it'll return `undefined` instead of `null` but if that's a problem (shouldn't be, cause react should dump the undefined, but in case) you can do `filterStore?._shade ?? null` and use both the shiny new operators.  Again - not saying you should change it, I'm just proselytizing the optional chain because it's about time we had an operator that saved all that boilerplate

Reply:
Wow, I didn't know that. Thats cool, a shortcut for a shortcut. I made that change, thanks for letting me know.